pytest_httpserver/pytest_plugin.py

```diff
@@ -26,34 +10,31 @@ def get_httpserver_listen_address():
     if listen_port:
         listen_port = int(listen_port)
 
-    return (listen_host, listen_port)
+    return listen_host, listen_port
 
 
-@pytest.fixture
+@pytest.fixture(scope="session")
 def httpserver_listen_address():
     return get_httpserver_listen_address()
 
 
-@pytest.fixture
-def httpserver(httpserver_listen_address):
-    if Plugin.SERVER:
-        Plugin.SERVER.clear()
-        yield Plugin.SERVER
-        return
-
+@pytest.fixture(scope="session")
+def _httpserver(httpserver_listen_address):
```

Comment:
Could you rename this fixture? I think it can be overridden later by the user, and that would add a great value to the library so this could be part of the public API, and documented so it shouldn't start with an underscore.

This could be `init_httpserver` or `make_httpserver` or `create_httpserver` something like that.